### PR TITLE
Fixed api locations route

### PR DESCRIPTION
**Pull request by cordoval**
|Q            |A                                             |
|---          |---                                           |
|Bug Fix?     |no                                            |
|New Feature? |no                                            |
|BC Breaks?   |no                                            |
|Deprecations?|no                                            |
|Tests Pass?  |attempt to move the parameter to the extension|
|Fixed Tickets|on top of rogergros PR                        |
|License      |MIT                                           |
                                                              

adaptation to move the parameter to an extension parameter

### CODE REVIEW

File: src/Elcodi/Bundle/GeoBundle/DependencyInjection/ElcodiGeoExtension.php [hunk 1]
```diff
@@ -87,6 +87,8 @@ protected function getParametrizationValues(array $config)
             "elcodi.entity.location.mapping_file" => $config['mapping']['location']['mapping_file'],
             "elcodi.entity.location.manager"      => $config['mapping']['location']['manager'],
             "elcodi.entity.location.enabled"      => $config['mapping']['location']['enabled'],
+
+            "elcodi.route.location_api.prefix"    => $config['location_api_prefix'],
```
Comment (mmoreram) [inline src/Elcodi/Bundle/GeoBundle/DependencyInjection/ElcodiGeoExtension.php]: `elcodi.route.location_api.prefix` => `elcodi.route.location_api_prefix`

Comment (cordoval) [inline src/Elcodi/Bundle/GeoBundle/DependencyInjection/ElcodiGeoExtension.php]: oh without the dot, ok, on it

Comment (mmoreram) [inline src/Elcodi/Bundle/GeoBundle/DependencyInjection/ElcodiGeoExtension.php]: ping @cordoval

**Comment by cordoval** [comment]
@mmoreram done

**Comment by mmoreram** [comment]
@cordoval The name has changed, so it should change all its usages as well, right? :)

**Comment by cordoval** [comment]
@mmoreram bump ready and here is the ripple effect in bamboo http://github.com/elcodi/bamboo/pull/374

**Pull request merged**